src/GuiBase/Viewer/Gizmo/Gizmo.hpp

```diff
@@ -103,6 +101,10 @@ class Gizmo
     /// \param mesh Except declaration type, must be equal to ro->getMesh();
     void addRenderObject( Engine::RenderObject* ro, const std::shared_ptr<Engine::Mesh>& mesh );
 
+    /// Generate a plain rendertechnique to draw the gizmo.
+    Engine::RenderTechnique* makeRenderTechnique( const std::string mtlName,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Function parameter 'mtlName' should be passed by const reference.](https://app.codacy.com/manual/STORM/Radium-Engine/pullRequest?prid=4401329)